src/multi-select/dropdown.tsx

```diff
@@ -125,6 +139,11 @@ const Dropdown = ({
   const toggleExpanded = () =>
     setExpanded(isLoading || disabled ? false : !expanded);
 
+  const handleClearSelected = (e) => {
+    e.stopPropagation();
+    contentProps["onChange"] && contentProps["onChange"]([]);
```

Comment:
you can use optional chaining and make it one liner `contentProps?.onChange()`

Reply:
@harshzalavadiya I removed the undefined check since onChange is a required Prop.